### DISCUSSION TITLE
g2plot 折线图最大值渲染有问题

### DISCUSSION THREAD

**Discussion by xujaibo** [antvis/G2Plot#3573] [Q&A] [answered]
在使用g2plot折线图的时候，最大值的折线图渲染异常。如附件图片3所示，数据最大值为9016.16，渲染的折线图的最高点已经超过右侧标定的最大值刻度9016.6。具体图表配置见附件图片1，后端数据见附件图片2
<img width="790" alt="配置" src="https://user-images.githubusercontent.com/45301701/217516357-05d3d074-bf20-441c-85a7-f8db66bda648.png">
<img width="860" alt="数据" src="https://user-images.githubusercontent.com/45301701/217516364-c5477621-136f-442a-8583-86faf2873250.png">
<img width="848" alt="最高点" src="https://user-images.githubusercontent.com/45301701/217516371-0a9b066b-097a-4b95-9620-9add8fd3e55d.png">

**Comment by visiky** [comment]
可以转化为一个codesandbox复现例子吗

**Comment by hustcc** [comment, marked as answer]
合理的，似乎没有什么问题，因为你的数据大，且数据之间的差距很小。这个图中，9016.16 > 9016.1，是对的。

![image](https://github.com/antvis/G2Plot/assets/7856674/e0cae44d-d1c1-4352-9421-9b3f31ac64be)

**Reply by hustcc** [reply]
你也可以设置一下图表 y 轴最小值从 0 开始，就会发现没问题。

```ts
meta: {
  value: { min: 0 }
}
```
